index.bs

```diff
@@ -22,6 +22,8 @@ spec:webidl; type:dfn; text:resolve
 <pre class=anchors>
 urlPrefix: https://tc39.github.io/ecma262/; spec: ECMA-262
   type: dfn; text: realm; url: realm
+urlPrefix: https://storage.spec.whatwg.org/; spec: storage
+  type: dfn; text: storage; url: site-storage
```

Comment:
What's this for? I don't see a `[=/storage=]` (etc) link

Reply:
This is a term storage used to provide, it is linked to from the privacy section `Directories where the user agent stores [=storage|website storage=]`.

Reply:
Ah, thanks. I missed that variant in my search!